test/end-to-end/validations.spec.js

```diff
@@ -69,7 +66,7 @@ describe('apply validations migration examples', function () {
       .on(/\? Do you want to apply the migration \(Y\/n\)/).respond('y\n')
       .expect(assert.plans.contentType.create('dieatary-food', { name: 'Dieatary Food', description: 'Food with up to 500 calories' }))
       .expect(assert.plans.field.create('name', { type: 'Symbol', name: 'name of the food', validations: [{ unique: true }] }))
-      .expect(assert.plans.field.create('calories', { type: 'Number', name: 'amount of calories the food contains', validations: [{ range: { 'max': 500 } }] }))
+      .expect(assert.plans.field.create('calories', { type: 'Link', linkType: 'Asset', name: 'amount of calories the food contains', validations: [{ assetImageDimensions: { width: { min: 1199, max: null }, height: { min: null, max: null } } }] }))
```

Comment:
It would be great if we have another test for the wrong case, for example, when one of the fields is omitted `{min: 1232}`